feature_flags.go

```diff
@@ -0,0 +1,80 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+// FeaturesService handles the communication with the application FeaturesService
+// related methods of the GitLab API.
+//
+// GitLab API docs:
+// https://gitlab.com/gitlab-org/gitlab-ce/blob/9-3-stable/doc/api/features.md
```

Comment:
Could you please update all the links so they point to `https://docs.gitlab.com/ce/api/...` instead of to a specific version of the docs.